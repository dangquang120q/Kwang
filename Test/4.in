9
88
3259 8246 7312 7743 7854 2101 2310 4397 3972 8394 6560 514 445 4626 8188 9175 3610 9287 1279 5007 1170 4139 8750 1707 6828 1615 1402 8648 5153 9559 2686 9539 9363 1673 9564 4288 2341 8574 8422 7069 3404 5284 4450 4503 165 7280 918 6458 6705 4711 8973 6506 2076 8668 2968 9639 9796 5604 5422 6282 1157 4835 4998 3578 86 2440 950 9935 301 5912 8362 3315 7029 9965 4290 8142 3648 8204 9536 9068 941 146 9272 6085 5231 3417 2075 3693 
6
7120 5277 7432 513 9550 1073 
51
7217 3221 2488 8291 2640 7915 807 7896 5076 6540 1310 4656 6532 1687 4642 3212 1038 7492 6699 8299 7191 6510 3332 3097 6522 7318 391 3412 7224 1527 9086 5867 2484 9698 9144 5593 7568 9181 9323 6967 6541 2976 4434 4331 8794 790 5518 8160 8444 3471 9400 
87
8707 2816 5533 6202 313 7493 410 2569 844 6302 9548 9976 9177 6889 28 6359 943 5153 5549 6302 114 2188 3417 5029 6351 1181 725 9021 5490 3899 2477 5633 5868 1231 3872 1457 4019 9998 6562 6934 1393 2252 2842 1993 6154 2348 7667 6513 6873 5336 7385 3088 8530 9863 8255 2530 268 4808 5870 4339 3962 9801 6601 9535 9687 2213 6830 4113 9304 7038 5410 858 3785 8915 6920 9005 6117 2593 8998 8819 9296 757 312 250 9106 9302 3893 
32
9981 585 5699 1419 8973 9432 1918 7211 3653 6154 11 1985 883 3801 6347 4719 4126 9808 6280 555 2427 8231 1290 2479 8872 2631 8447 9959 3580 6284 9722 2310 
31
2347 6560 8057 7584 6763 2771 7891 1854 5357 2970 3421 4633 1741 564 7790 7264 2026 7289 6674 9503 2964 2409 8160 7747 7212 4198 5930 6666 9312 760 6623 
62
6826 113 1015 2884 774 4918 8222 9024 2537 4794 8488 9527 8939 8248 2434 6647 1902 3149 1901 8643 5872 3708 585 1655 5356 1087 9594 1715 8436 5790 1327 6154 7184 8886 7154 4348 8993 7927 4432 2074 1593 6184 4923 5807 9933 3143 3321 5156 5867 7531 7549 1115 2333 9517 5771 7901 4874 1558 9935 8206 1035 2550 
67
7143 8983 3064 8728 9177 6516 4902 3773 9819 9367 1946 1331 4298 1903 7953 1443 7434 86 6534 7427 4501 7828 1029 7649 6215 2179 1386 4154 2382 849 4844 6644 8649 2256 8984 5980 5393 9835 3557 8752 8334 7620 1147 5835 3141 7490 244 2563 8406 8594 3090 9214 541 7188 275 678 8952 2457 7186 8813 8410 1176 950 2009 6475 4276 5307 
16
764 7191 4219 2238 3603 5161 2227 1203 8878 9845 8771 2810 9236 356 3312 4374 
