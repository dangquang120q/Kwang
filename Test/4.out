29696
24102
28421
29837
29748
26975
29462
29021
27959
